docs/src/main/sphinx/installation/cli.rst

```diff
@@ -381,10 +383,39 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
+Batch mode
+----------
+
+Running the Trino CLI with the ``--execute``, ``--file`` or passing queries to
+the standard input uses the batch mode (non-interactive), in which
```

Comment:
uses the batch (non-interactive) mode. In this mode the CLI does not report progress, and exists after processing the supplied queries. Results are printed in ``CSV`` format by default. You can configure other formats and redirect the output to a file.

Reply:
I think we should add an example for that redirection/piping into a file...

Reply:
That example varies across shells and operating systems combinations. Let's leave it to the shell's documentation instead IMO.